api_tests/logs/views/test_log_detail.py

```diff
@@ -141,3 +141,12 @@ def test_title_hidden_from_non_contributor_in_file_move(self):
         assert_equal(res.status_code, 200)
         assert_not_in(self.component.title, res.json['data'])
         assert_equal(res.json['data'][0]['attributes']['params']['source']['node_title'], 'Private Component')
+
+    def test_file_params(self):
+        res = self.app.get(self.node_logs_url, auth=self.user_one.auth)
+        assert_equal(res.status_code, 200)
+        assert_not_equal(
```

Comment:
This seems like only half a test. Shouldn't it equal something predictable as well?